app/code/Magento/Wishlist/Test/Unit/Controller/Shared/AllcartTest.php

```diff
@@ -28,19 +28,19 @@ class AllcartTest extends TestCase
     protected $allcartController;
 
     /**
-     * @var \Magento\Framework\TestFramework\Unit\Helper\ObjectManager
+     * @var ObjectManagerHelper
      */
     protected $objectManagerHelper;
 
     /**
-     * @var Context
+     * @var WishlistProvider|MockObject
      */
-    protected $context;
+    protected $wishlistProviderMock;
 
     /**
-     * @var WishlistProvider|MockObject
+     * @var Context|MockObject
      */
-    protected $wishlistProviderMock;
+    protected $context;
```

Comment:
Please revert changing property position, also would be good to make all property private **in tests**